app/views/admin/grove_monitor/_student.html.erb

```diff
@@ -1,10 +1,16 @@
-<div class="col-xs-3">
-  <div class="thumbnail">
-    <img src="..." alt="...">
-    <div class="caption">
-      <h3><%= student.name if student %></h3>
-      <p>...</p>
-      <p><a href="#" class="btn btn-primary" role="button">Button</a> <a href="#" class="btn btn-default" role="button">Button</a></p>
+<% if student %>
```

Comment:
There shouldn't be a need for this `if` statement in the student partial. There should be a student if this is being rendered.